src/dataflow/src/source/mod.rs

```diff
@@ -556,8 +564,8 @@ impl SourceMetrics {
             )
             .unwrap();
             static ref CAPABILITY: UIntGaugeVec = register_uint_gauge_vec!(
-                "mz_kafka_capability",
-                "The current capability for this dataflow. This corresponds to min(mz_kafka_partition_closed_ts)",
+                "mz_capability",
+                "The current capability for this dataflow. This corresponds to min(mz_partition_closed_ts)",
                 &["topic", "source_id", "worker_id"]
```

Comment:
Does `topic` make sense for every kind of source? I think, but am open to other ideas, that it makes more sense to leave this as `mz_kafka_capability` and add other metrics that have more specific labels (e.g. kinesis, file, etc.